app/controllers/admin/divisions_controller.rb

```diff
@@ -112,7 +112,9 @@ def index_filter
   end
 
   def prep_form_vars
-    @currency_choices = Currency.order(:name)
+    @currency_choices = Currency.order(:name).map do |currency|
+      helpers.currency_name(currency, count: 1)
```

Comment:
I'm not sure about this approach, this feels like a bit of a code smell to me. Maybe we can figure this out next week? This story wasn't scheduled for this sprint anyway.

Reply:
ok

found this btw
https://blog.bigbinary.com/2016/06/26/rails-add-helpers-method-to-ease-usage-of-helper-modules-in-controllers.html